apps/mull-ui/src/app/access-token.ts

```diff
@@ -0,0 +1,9 @@
+let accessToken: string | null = null;
```

Comment:
Should this maybe be part of the user context? Why did you remove it from there

Reply:
I removed it since it needs to be accessible outside the React Application. Mainly due to the fact that it needs to be used by Apollo to set the authorization header.